### DISCUSSION TITLE
Create own scene?

### DISCUSSION THREAD

**Discussion by Doev** [NVlabs/instant-ngp#458] [General]
please delete

**Comment by mottosso** [comment]
No please don't delete, I had the exact same question and I bet others have too. 😅 

The README does a great job explaining how to get from start to running the examples, but has hidden the actual usage instructions behind a tiny little link somewhere in the middle of the page.

---

<img width="659" alt="image" src="https://user-images.githubusercontent.com/2152766/167714025-e0ff880a-6877-469a-a7a2-fc2f364ba5c4.png">

---

Can you spot it? 😄 

- https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md

To the authors, I would suggest putting this front-and-center, even ahead of building instructions such that it cannot be missed. 👍

**Reply by Tom94** [reply]
You're right -- I made it more prominent! (Although not front-and-center, given that the intent of the codebase is to be more general than just NeRF.)
